### ISSUE TITLE
Question on Decoupling Representation Learning from Reinforcement Learning: Why use MoCo instead of BYOL and Why use W in loss instead of architecture

### ISSUE THREAD

**Issue by SohamTamba** [astooke/rlpyt#190]
Hey guys, I was going through your work [here](https://arxiv.org/abs/2009.08319). Congratulations on beating end-to-end RL 😄. 

I wanted to ask two questions:
1. The Self supervised training architecture is seems to based on MoCo - except it uses future frames for positive pairs -, but BYOL achieves better results than MoCo, without negative sampling. Did you guys consider pretraining the encoder using BYOL?
2. I noticed that the transformation matrix W is used in the loss function. Is there any reason it was not used in the architecture as a linear layer? i.e. instead off: _L = p^T * W * c_ , why not _c' = W * c_ , _L = cosine_similarity(p, c')_. Is _W_ trainable?
